src/Sets/Tetrahedron.jl

```diff
@@ -97,11 +97,27 @@ Check whether a given point is contained in a tetrahedron.
 
 ### Algorithm
 
-Currently falls back to the `VPolytope` implementation.
-TODO Implement efficient version, see LazySets#3259.
+For each plane of the tetrahedron, we check if the point `x` is on the same side as the remaining vertex.
+We need to check this for each plane [1].
+
+[1] https://stackoverflow.com/questions/25179693/how-to-check-whether-the-point-is-in-the-tetrahedron-or-not
 """
 function ∈(x::AbstractVector, T::Tetrahedron)
-    return x ∈ convert(VPolytope, T)
+    v = T.vertices
+    return same_side(v[1], v[2], v[3], v[4], x) &&
+           same_side(v[4], v[1], v[2], v[3], x) &&
+           same_side(v[3], v[4], v[1], v[2], x) &&
+           same_side(v[2], v[3], v[4], v[1], x)
+end
+
+"""
+Return `true` iff point `x` lies in the same half-space as `v4` with respect to the hyperplane determined by `v1`, `v2` and `v3`.
+"""
+function same_side(v1, v2, v3, v4, x)
```

Comment:
I think you need to use normal comments because `Documenter` may read this as a documentation (which is not included).